README.windows.md

```diff
@@ -45,9 +45,9 @@ or edit `%USERPROFILE%\.gitconfig` and add/edit the lines:
 # Source distribution
 
 ## Supported build platforms
-- Windows 10: supported (32 and 64 bits)
-- Windows 8: supported (32 and 64 bits)
-- Windows 7: supported (32 and 64 bits)
+- Windows 10: supported (32- and 64-bits)
+- Windows 8: supported (32- and 64-bits)
+- Windows 7: supported (32- and 64-bits)
```

Comment:
Right, space and bits is not incorrect, but I meant to drop the s from
bits, for consistency..

2015-08-23 19:00 GMT+00:00 Stefan Karpinski <notifications@github.com>:

> In README.windows.md
> <https://github.com/JuliaLang/julia/pull/12769#discussion_r37709361>:
>
> > @@ -45,9 +45,9 @@ or edit `%USERPROFILE%\.gitconfig` and add/edit the lines:
> >  # Source distribution
> >
> >  ## Supported build platforms
> > -- Windows 10: supported (32 and 64 bits)
> > -- Windows 8: supported (32 and 64 bits)
> > -- Windows 7: supported (32 and 64 bits)
> > +- Windows 10: supported (32- and 64-bits)
> > +- Windows 8: supported (32- and 64-bits)
> > +- Windows 7: supported (32- and 64-bits)
>
> This is actually correct without the hyphen since it is "32 bits"; you
> could also write "32-bit" which is an adjective.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/JuliaLang/julia/pull/12769/files#r37709361>.
>



-- 
Palli.